packages/env-utils/src/env.ts

```diff
@@ -0,0 +1,45 @@
+import UAParser from 'ua-parser-js';
```

Comment:
nit: I would probably name this file just `utils.ts`... or `envUtils.ts` to make it easier findable quick find file.